plugins/kubeval.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+￼kind: Plugin
+￼metadata:
+￼  name: kubeval
+￼spec:
+￼  version: "0.14.0"
+￼  shortDescription: Validate schema of resource files
+  description: |
+    The plugin validates a Kubernetes YAML or JSON configuration file.
+    It does so using schemas generated from the Kubernetes OpenAPI specification,
+    and therefore can validate schemas for multiple versions of Kubernetes
+  homepage: https://github.com/instrumenta/kubeval
```

Comment:
The indentation of these two fields is wrong. Please fix this to get the CI pass.